### PR TITLE
feat(onramp-kit): Allow to send default options to start the stripe sessions

### PR DESCRIPTION
**Pull request by yagopv**
## What it solves

Currently we only allowed to start the stripe session specifying the networks and the wallet. The stripe API support [more options](https://stripe.com/docs/crypto/using-the-api) so we are going to allow to passthrough any of them to start the session in the server

### CODE REVIEW

**Review by dasanra — CHANGES_REQUESTED** [review]
Tests are failing

**Comment by yagopv** [comment]
> Tests are failing

fixed

File: packages/onramp-kit/src/SafeOnRampKit.ts [hunk 1]
```diff
@@ -1,59 +1,72 @@
...
+   * @param handler The handler to be called when the event is triggered
+   */
+  subscribe(event: SafeOnRampEvent<TAdapter>, handler: SafeOnRampEventListener<TAdapter>) {
+    this.#adapter.subscribe(event, handler)
+  }
+
+  /**
+   * Unsubscribe from provider events
+   * @param event The specific event to unsubscribe from
+   * @param handler Tje handler to be removed from the event
```
Comment (germartinez) [inline packages/onramp-kit/src/SafeOnRampKit.ts:67]:
```suggestion
   * @param handler The handler to be removed from the event
```

**Review by germartinez — APPROVED** [review]

File: packages/onramp-kit/src/SafeOnRampKit.ts (hunk 1 above)
Comment (germartinez) [reply packages/onramp-kit/src/SafeOnRampKit.ts:67]: Too late :(

**Pull request merged**